src/library_sdl.js

```diff
@@ -1946,6 +1946,7 @@ var LibrarySDL = {
     } else {
       delete surfData.clipRect;
     }
+    return 1;
```

Comment:
I don't really know.   Perhaps I will revert this line and just continue to return undefined/0.. to avoid having to figure it out.  In any case the native signature returns SDL_BOOL.

Reply:
Reverted